src/main/java/cinema/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,45 @@
+package cinema.controller;
+
+import cinema.model.dto.shoppingcart.ShoppingCartResponseDto;
+import cinema.service.MovieSessionService;
+import cinema.service.ShoppingCartService;
+import cinema.service.UserService;
+import cinema.service.mapper.ShoppingCartMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/shopping-carts")
+public class ShoppingCartController {
+    private final ShoppingCartService shoppingCartService;
+    private final UserService userService;
+    private final MovieSessionService movieSessionService;
+    private final ShoppingCartMapper shoppingCartMapper;
+
+    @Autowired
+    public ShoppingCartController(ShoppingCartService shoppingCartService,
+                                  UserService userService,
+                                  MovieSessionService movieSessionService,
+                                  ShoppingCartMapper shoppingCartMapper) {
+        this.shoppingCartService = shoppingCartService;
+        this.userService = userService;
+        this.movieSessionService = movieSessionService;
+        this.shoppingCartMapper = shoppingCartMapper;
+    }
+
+    @PostMapping("/movie-sessions")
+    public void addMovieSession(@RequestParam("userId") Long userId,
+                                @RequestParam("movieSessionId") Long sessionId) {
```

Comment:
```suggestion
                                @RequestParam("movieSessionId") Long sessionId) {
```
but here the names are different so it is ok